modules/prebidServerBidAdapter.js

```diff
@@ -196,10 +198,15 @@ function PrebidServer() {
             bidObject.creative_id = bidObj.creative_id;
             bidObject.bidderCode = bidObj.bidder;
             bidObject.cpm = cpm;
-            bidObject.ad = bidObj.adm;
-            if (bidObj.nurl) {
-              bidObject.ad += utils.createTrackPixelHtml(decodeURIComponent(bidObj.nurl));
+            if (bidObj.media_type === 'video') {
+              bidObject.vastUrl = bidObj.nurl;
+            } else {
+              bidObject.ad = bidObj.adm;
+              if (bidObj.nurl) {
+                bidObject.ad += utils.createTrackPixelHtml(decodeURIComponent(bidObj.nurl));
+              }
```

Comment:
Looks like these should be the other way around. From the spec, `adm` is described as:

> Optional means of conveying ad markup in case the bid wins; supersedes the win notice if markup is included in both. 

(see section 4.2.3)